test/functional/bip68-sequence.py

```diff
@@ -21,15 +21,11 @@ def __init__(self):
         super().__init__()
         self.num_nodes = 2
         self.setup_clean_chain = False
+        self.extra_args = [[], ["-acceptnonstdtxn=0"]]
 
-    def setup_network(self):
-        self.nodes = []
-        self.nodes.append(start_node(0, self.options.tmpdir))
-        self.nodes.append(start_node(1, self.options.tmpdir, ["-acceptnonstdtxn=0"]))
+    def run_test(self):
         self.relayfee = self.nodes[0].getnetworkinfo()["relayfee"]
-        connect_nodes(self.nodes[0], 1)
```

Comment:
Also, since setup_nodes isn't called here, doesn't this need to be called at some point?